gtest/src/manager.rs

```diff
@@ -446,6 +450,61 @@ impl ExtManager {
 
         core_processor::handle_journal(journal, self);
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn execute_custom_function(
```

Comment:
Yeah, I see that you created universal functionality for calling any custom function, BUT:
for real production we will apply such changes directly in main executors

so for gtest we should provide only `call_meta(name)`
where name is fn with no-void return type

also executing "name" should never affect storage: there is no need to handle journal after this.